datagovharvester/extract/dcatus.py

```diff
@@ -0,0 +1,26 @@
+from .utils import fetch_url
+import json
+from datagovharvester.utils.s3_utilities import (
+    upload_to_S3,
+    create_s3_payload,
+)
+
+
+def extract_dcatus_catalog(url, job_id, S3_client, bucket_name):
```

Comment:
I might be missing something, but are we making this specific to `dcatus` because we anticipate making this function more `dcatus` specific?

Reply:
The only `dcatus`-ness in the function is the `dataset` key access and the json-stringify of each record/object in the `dataset` array ( i.e. `json.dumps(record)` ). Besides that everything else should be format agnostic...